one-ui/src/main/java/com/marklogic/hub/oneui/auth/HttpHandshakeInterceptor.java

```diff
@@ -0,0 +1,35 @@
+package com.marklogic.hub.oneui.auth;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.http.server.ServerHttpRequest;
+import org.springframework.http.server.ServerHttpResponse;
+import org.springframework.http.server.ServletServerHttpRequest;
+import org.springframework.http.server.ServletServerHttpResponse;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.web.socket.WebSocketHandler;
+import org.springframework.web.socket.server.HandshakeInterceptor;
+
+public class HttpHandshakeInterceptor  implements HandshakeInterceptor {
```

Comment:
How is this used? I don't see any references to it anywhere. I'm also wondering what problem it's solving, as I don't think DHFPROD-4337 spells it out? This may be needed, but if so, we should have a JIRA ticket that spells out the problem that this is solving.

Reply:
I accidently checked in the file. It was created on my local for some testing. We do not need. it.